wcfsetup/install/files/acp/templates/applicationManagement.tpl

```diff
@@ -14,42 +14,116 @@
 	{/hascontent}
 </header>
 
-<div class="section tabularBox">
-	<table class="table">
-		<thead>
-			<tr>
-				<th class="columnID columnPackageID" colspan="2">{lang}wcf.global.objectID{/lang}</th>
-				<th class="columnText columnPackageName">{lang}wcf.acp.package.name{/lang}</th>
-				<th class="columnText columnDomainName">{lang}wcf.acp.application.domainName{/lang}</th>
-				<th class="columnText columnDomainPath">{lang}wcf.acp.application.domainPath{/lang}</th>
-				<th class="columnText columnLandingPageID">{lang}wcf.acp.application.landingPage{/lang}</th>
+{if $isMultiDomainSetup}
+	<div class="warning">{lang}wcf.acp.application.multiDomain{/lang}</div>
+{/if}
+
+<form method="post" action="{link controller='ApplicationManagement'}{/link}">
+	{if !$isMultiDomainSetup}
+		<section class="section">
+			<h2 class="sectionTitle">{lang}wcf.acp.application.management.domain{/lang}</h2>
+
+			<dl{if $errorField == 'domainName'} class="formError"{/if}>
+				<dt><label for="domainName">{lang}wcf.acp.application.management.domainName{/lang}</label></dt>
+				<dd>
+					<div class="inputAddon">
+						<span class="inputPrefix">http(s)://</span>
+						<input type="text" name="domainName" id="domainName" value="{$domainName}" class="long">
+					</div>
+					{if $errorField == 'domainName'}
+						<small class="innerError">
+							{if $errorType == 'empty'}
+								{lang}wcf.global.form.error.empty{/lang}
+							{else}
+								{lang}wcf.acp.application.management.domainName.error.{$errorType}{/lang}
+							{/if}
+						</small>
+					{/if}
+					<small>{lang}wcf.acp.application.management.domainName.description{/lang}</small>
+				</dd>
+			</dl>
+
+			<dl{if $errorField == 'cookieDomain'} class="formError"{/if}>
+				<dt><label for="cookieDomain">{lang}wcf.acp.application.management.cookieDomain{/lang}</label></dt>
+				<dd>
+					<input type="text" name="cookieDomain" id="cookieDomain" value="{$cookieDomain}" class="long">
+					{if $errorField == 'cookieDomain'}
+						<small class="innerError">
+							{if $errorType == 'empty'}
+								{lang}wcf.global.form.error.empty{/lang}
+							{else}
+								{lang}wcf.acp.application.management.cookieDomain.error.{$errorType}{/lang}
+							{/if}
+						</small>
+					{/if}
+					<small>{lang}wcf.acp.application.management.cookieDomain.description{/lang}</small>
+				</dd>
+			</dl>
+		</section>
+
+		<script>
+			(() => {
+				const domainName = document.getElementById("domainName");
+				const cookieDomain = document.getElementById("cookieDomain");
+
+				// Keep the cookie domain in sync if it was previously identical.
+				if (domainName.value === cookieDomain.value) {
```

Comment:
It's a bit odd that this is checked in JavaScript, instead of only injecting this piece of JavaScript when the values differ using PHP.